apps/hash-frontend/sentry.edge.config.ts

```diff
@@ -0,0 +1,12 @@
+import * as Sentry from "@sentry/nextjs";
```

Comment:
Sentry now works in the Edge runtime.

These configs have to be specified separately because the client config's use of `Replay` is not valid in server contexts.